dev-libs/aws-c-common/aws-c-common-0.4.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="Core c99 package for AWS SDK for C"
+HOMEPAGE="https://github.com/awslabs/aws-c-common"
+SRC_URI="https://github.com/awslabs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+
+DEPEND=""
```

Comment:
You can remove these empty variables from the ebuild if you wish so.

Reply:
Check!